src/sqlitestorage.h

```diff
@@ -241,8 +241,11 @@ class MKCAL_EXPORT SqliteStorage : public ExtendedStorage
     Private *const d;
     //@endcond
 
-public Q_SLOTS:
-    void fileChanged(const QString &path);
+public slots:
```

Comment:
Even if the sqlite backend is considered more private it wouldn't be bad to keep Q_SLOTS on library repository. Comes quite cheap anyway and can avoid problems when needing to mix qt and glib code.